components/HomePage/InfoSection.tsx

```diff
@@ -11,7 +11,7 @@ export default function InfoSection({ imgStart }: InfoSectionProps) {
         <InfoWrapper>
           <InfoRow imgStart={imgStart}>
             <Column>
-              <TextWrapper>
+              <div>
```

Comment:
Doesn't this area need any context? If so, you might consider using `section` or `article`